Automattic/wp-calypso
# Design Picker cleanup: Remove unused SiteSetupDesignPicker

#### Proposed Changes

As we are now using `UnifiedDesignPicker` for `/setup` flow, this PR removes the old `SiteSetupDesignPicker` component which is unused.

This PR also removes the following flags which are already `true` anywhere:

- `signup/design-picker-unified`
- `signup/design-picker-generated-designs`

#### Testing Instructions

* Go to `/setup`
* Test happy cases of the Design Picker flows
* Verify that nothing breaks :)

#### Pre-merge Checklist






Related to #68244
